tests/handlers/test_model_repository.py

```diff
@@ -101,11 +101,7 @@ async def test_load_removes_stale_models(
 
     # Validate that the stale test models have been loaded
     registry_models = await model_registry.get_models(sum_model_settings.name)
-    stale_length = (
-        len(to_load)
-        + 1  # Count the (stale) default model
-        + 1  # Count the previous (non-stale) model
-    )
+    stale_length = len(to_load) + 1  # Count the previous (non-stale) model
```

Comment:
I think the previous test case was correct, and was just flagging that the default model is no longer present - this may be because it doesn't have a version, therefore it's not present on the `self._versions` dict (see my other comment above).